homeassistant/components/reolink/host.py

```diff
@@ -150,6 +155,14 @@ async def async_init(self) -> None:
                 f"a-z, A-Z, 0-9 or {ALLOWED_SPECIAL_CHARS}"
             )
 
+        store: Store[str] | None = None
+        if self._config_entry_id is not None:
+            store = get_store(self._hass, self._config_entry_id)
+            if self._config.get(CONF_SUPPORTS_PRIVACY_MODE):
+                data = await store.async_load()
+                if data:
```

Comment:
Challenge for a future PR, squash it into a single if-statement with walrusses :fishsticks:

Reply:
Thanks, challenge accepted, follow up PR: https://github.com/home-assistant/core/pull/137069